app/views/upload.scala.html

```diff
@@ -65,7 +65,7 @@ <h2 class="govuk-error-summary__title logged-out-title" id="logged-out-error-tit
                                 tabindex="-1" role="alert" aria-describedby="success-message-text">
                                 <div>
                                     @greenTickMark()
-                                    <p id="success-message-text" >@Html(Messages("upload.dragAndDropSuccessMessage"))</p>
+                                    <p id="success-message-text">The folder "<span id="folder-name"></span>" (containing <span id="folder-size"></span> has been selected</p>
```

Comment:
Looks like there's a missing closing bracket `)` after the closing `</span>`.

Reply:
Done!